minio/api.py

```diff
@@ -1286,7 +1286,8 @@ def remove_incomplete_upload(self, bucket_name, object_name):
                 return
 
     def presigned_get_object(self, bucket_name, object_name,
-                             expires=timedelta(days=7)):
+                             expires=timedelta(days=7),
+                             response_headers=None):
```

Comment:
1. You could name the param as `custom_headers`
2. You could have pythonic way of emptiness like `response_headers={}`